YSE_App/data_utils.py

```diff
@@ -1511,67 +1519,102 @@ def rm_frb_galaxy(request):
-
-      - transient_name (str): Name of the FRBTransient object
-        Must be in the DB already
-      - table (str): a table of the PATH candidates and their 
-        PATH results, stored as JSON
-      - F (str): name of filter; must be present in the
-        PhotometricBand table
-      - instrument (str): name of the instrument; must be present in the
-        Instrument table
-      - obs_group (str): name of the instrument; must be present in the
-        ObservationGroup table
-      - P_Ux (float): Unseen posterior;  added to the transient
-      - bright_star (int): 1 if the transient is near a bright star
-
-    Args:
-        request (requests.request): 
-            Request from outside FFFF-PZ
-
-    Returns:
-        JsonResponse: 
+    Debug endpoint to print out the incoming request method, headers, and body.
+    No JSON parsing, no assumptions.
     """
-    
-    # Parse the data into a dict
-    data = JSONParser().parse(request)
+    print("\n=== DEBUGGING NEW REQUEST ===")
+    print(f"Request Method: {request.method}")
+    print(f"Request Content-Type: {request.content_type}")
+    print(f"Request Headers:")
+    for k, v in request.headers.items():
+        print(f"    {k}: {v}")
+    print(f"Raw Request Body: {request.body}")
+
+    return JsonResponse({"message": "Request debugged successfully."})
+
+@method_decorator(csrf_exempt, name='dispatch')
+class IngestPathView(APIView):
+    """
+    API endpoint for ingesting PATH results, with full debug output.
+    """
+    authentication_classes = [BasicAuthentication]
+    permission_classes = [IsAuthenticated]
 
-    # Deal with credentials
-    auth_method, credentials = request.META['HTTP_AUTHORIZATION'].split(' ', 1)
-    credentials = base64.b64decode(credentials.strip()).decode('utf-8')
-    username, password = credentials.split(':', 1)
-    user = auth.authenticate(username=username, password=password)
+    def post(self, request, format=None):
+        return self.handle_ingestion(request)
 
-    try:
-        itransient = FRBTransient.objects.get(name=data['transient_name'])
-    except:
-        return JsonResponse({"message":f"Could not find transient {data['transient_name']} in DB"}, status=400)
-    # Prep 
-    tbl = pandas.read_json(data['table'])
+    def put(self, request, format=None):
+        return self.handle_ingestion(request)
 
-    try:
-        path.ingest_path_results(
-            itransient, tbl, 
-            data['F'], 
-            data['instrument'], data['obs_group'],
-            data['P_Ux'], user,
-            remove_previous=True,
-            bright_star=data['bright_star']) # May wish to make this optional
-    except:
-        print("Ingestion failed")
-        return JsonResponse({"message":f"Ingestion failed 2x!"}, status=405)
-    else:
-        print("Successfully ingested")
+    def handle_ingestion(self, request):
+        try:
+            # === DEBUG SECTION ===
+            print("========== DEBUG START ==========")
```

Comment:
do we want all of these print statements commented out, or removed or put behind a flag?